src/test/common.ts

```diff
@@ -59,3 +61,39 @@ export function isBrowser(browsers: Browser | Browser[], ua = window.navigator.u
     return browsersStr.indexOf(browser) > -1;
   }
 }
+
+export interface FakeEvent extends Event {
+  which: number;
+  keyCode: number;
+  key: string;
+}
+
+export function createKeyboardEvent(input): Event {
```

Comment:
This seems useful but we should demonstrate this by actually using this helper in the existing test(s).